### ISSUE TITLE
DAI in DSR multiplies every time I refresh blockchain balances

### ISSUE THREAD

**Issue by 4eed8ack** [rotki/rotki#1110]
## Problem Definition

As described above, my 10 DAI in Oasis Saving turn into 20 DAI, then 30 DAI, and so on every time I hit refresh on blockchain balance.

### System Description

Running Appimage Rotki on Fedora 32.

**Comment by isidorosp** [comment]
Could you provide a little more information?
* Does the balance revert to the "correct" balance if you browse away from the dashboard and back to the dashboard, or does it stay incorrect?
* Does the balance revert to the "correct" balance if you restart the application?
* If you go to the "Accounts & Balances" page and then scroll to the "Blockchain Balances per Asset" balances, is the DAI balance matching what you see in the dashboard or is it "correct"?
* If you go to the "Accounts & Balances" page and then scroll down to your "ETH per account balances" balances, is the account that that DAI is in also being inflated?

**Comment by 4eed8ack** [comment]
Of course! My bad!

It only reverts to the correct amount after I quit the application then restart it/log in. Browsing away or to Dashboard and Account & Balances still shows the incorrect, multiplied amount of DAI. And my ETH and every other Token balance stays as the correct amount throughout the process.

TL;DR  1. Stays incorrect  2.Yes  3. Matches what I see in the dashboard (incorrect amount) 4. Yes, also incorrect amount.

**Comment by isidorosp** [comment]
No need to apologize! Thanks for the quick follow-up, it really helps.

It looks like the issue is in the frontend which means that your saved balances are not incorrect -- which is great!. We will try to reproduce and fix ASAP.

**Comment by LefterisJP** [comment]
We checked a bit and it may also be in the backend. Since the amount also seems to get appended again in the network tab. Will take a deeper look and hopefully solve it today.

**Comment by LefterisJP** [comment]
Hey @4eed8ack we fixed this issue in the latest develop branch. It will be included in the next version. Either v1.5.1 or v1.6.0.

**Comment by 4eed8ack** [comment]
Sounds great! Thank you very much, and I will looking out for the update.
